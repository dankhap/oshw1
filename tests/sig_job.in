infloop.c
(@)^z
jobs
infloop2.c
(@)^z
fg
(@)^z
jobs
fg 2
(@)^c
jobs
fg
(@)^c
infloop.c&
jobs
(@)^c