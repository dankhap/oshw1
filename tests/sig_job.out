[1] ./infloop.out : 22423 0 secs (Stopped)
./infloop.out
[1] ./infloop.out : 22423 -1 secs (Stopped)
[2] ./infloop2.out : 22424 -1 secs (Stopped)
./infloop2.out
[1] ./infloop.out : 22423 -1 secs (Stopped)
[2] ./infloop.out : 22423 -1 secs
